Neos.Flow/Classes/ResourceManagement/ResourceManager.php

```diff
@@ -608,6 +603,35 @@ protected function initializeCollections()
         }
     }
 
+    /**
+     * Sanitize file content and remove content that is suspicious
+     * for now this only checks svg file to solve the issue here https://nvd.nist.gov/vuln/detail/CVE-2023-37611
+     *
+     * @todo create a feature from this and allow to register code for sanitizing file content before importing
+     */
+    protected function sanitizeImportedFileContent(string $filename, string $content): string
+    {
+        $typeFromFilename = MediaTypes::getMediaTypeFromFilename($content);
+        $typeFromContent = MediaTypes::getMediaTypeFromFileContent($content);
+        if ($typeFromFilename === 'image/svg+xml' || $typeFromContent === 'image/svg+xml') {
+            // @todo: Simplify again when https://github.com/darylldoyle/svg-sanitizer/pull/90 is merged and released.
+            $previousXmlErrorHandling = libxml_use_internal_errors(true);
+            $sanitizer = new Sanitizer();
+            $sanitizedContent = $sanitizer->sanitize($content);
+            libxml_clear_errors();
+            libxml_use_internal_errors($previousXmlErrorHandling);
+            $issues = $sanitizer->getXmlIssues();
+            if ($issues && count($issues) > 0) {
+                if ($sanitizedContent === false) {
+                    throw new Exception('Sanitizing of suspicious file "' . $filename . '" failed during import.', 1695395560);
+                }
+                $content = $sanitizedContent;
+                $this->logger->debug(sprintf('Imported file "%s" contained suspicious content and was sanitized.', $filename), $issues);
```

Comment:
debug log might be too silent i think? How about at least an info?

```suggestion
                $this->logger->info(sprintf('Imported file "%s" contained suspicious content and was sanitized.', $filename), $issues);
```

Reply:
You are right, i think that justifies a warning